charts/hdfs-k8s/README.md

```diff
@@ -0,0 +1,63 @@
+HDFS `namenode` and `datanodes` running inside a kubernetes cluster.
+
+### Prerequisite
+
+  Requires Kubernetes version 1.5 and beyond, because `namenode` is using
+  `StatefulSet`, which is available only in version 1.5 and later.
+
+### Usage
+
+  1. Attach a label to one of your k8s cluster host that will run the `namenode`
+     daemon.
+
+  ```
+  $ kubectl label nodes YOUR-HOST hdfs-namenode-selector=hdfs-namenode-0
+  ```
+
+  2. Find the IP of your `kube-dns` name server that resolves pod and service
+     host names in your k8s cluster. Default is 10.96.0.10. It will be supplied
+     below as the `clusterDnsIP` parameter. Try this command and find the IP
```

Comment:
Do we need this configuration at all for `clusterDnsIP`? If we do, we should be using that of the service fronting kube-dns. You can get that IP address through `kubectl get svc --all-namespaces | grep dns`.

The individual kube-dns pods can get evicted and change their IP addresses.